Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -132,6 +132,8 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
 
         IMixedRealityInputSource RequestNewGenericInputSource(string name, IMixedRealityPointer[] pointers = null, InputSourceType sourceType = InputSourceType.Other);
 
+        BaseGlobalInputSource RequestNewGlobalInputSource(string name, IMixedRealityFocusProvider focusProvider = null, InputSourceType sourceType = InputSourceType.Other);
```

Comment:
I guess I missed this the first time around...a breaking change to a core interface makes me a little more hesitant, but I still think this is an overall good change.

Reply:
Also it returns a non-interface type :/